api/urls.py

```diff
@@ -12,7 +12,7 @@
 # Additionally, we include login URLs for the browsable API.
 urlpatterns = [
     path('', include(router.urls)),
-    path('meetings2/',views.MeetingsList.as_view()),
```

Comment:
@chriswedgwood  I noticed that this URL doesn't appear to even be used, so maybe you want to reject this change.

I have deleted path `/meetings2`

I have created path `/meetings` and mapped it to `views.MeetingsList.as_view()`

Reply:
Yes these needed a big tidy 👍